CMakeLists.txt

```diff
@@ -303,6 +307,11 @@ target_link_libraries(lovr
   ${LOVR_EMSCRIPTEN_FLAGS}
 )
 
+if (WIN32)
+  set_target_properties(lovr PROPERTIES LINK_FLAGS "/SUBSYSTEM:WINDOWS")
+  set_target_properties(lovr PROPERTIES LINK_FLAGS "/ENTRY:mainCRTStartup")
+endif()
+
```

Comment:
These can be removed now, I've got those added to master (with correct flags for debug/release).